src/components/dac_dataset_table/DACDatasetTableCellData.jsx

```diff
@@ -72,38 +72,9 @@ export function dataCustodianCellData({dataset, label = 'dataCustodianCellData'}
 }
 
 export function dataUseCellData({dataset, label = 'dataUseCellData', divClass = style['cell-data'], spanClass = style['data-use'], cellWidth = styles.cellWidths.dataUse, tooltipPlace = 'right'}) {
-  const codesAndDescriptions = dataset.dataUse?.primary ? dataset.dataUse.primary.map((dataUse) => {
```

Comment:
split these out into DataUseUtils in order to re-use them in the DatasetStatistics component